app/styles/tasks.css

```diff
@@ -273,6 +275,29 @@ input[type=range]:focus::-ms-fill-upper {
   margin-right: 0.5rem;
 }
 
+/* fetch task styles */
+
+.fetch-task {
+  margin-top: 1rem;
+}
+
+.fetch-task > p {
+  margin-bottom: 1rem;
+  font-weight: 700;
+}
+
+.fetch-task > button {
+  background-color: #008000;
+  padding: 1rem 5rem;
+  color: #ffffff;
+  font-size: 1rem;
+  border-radius: 50px;
+  cursor: pointer;
+}
+
+.fetch-task > button:disabled {
+  background-color: grey;
```

Comment:
Can we please have color variables for this?
Please follow the existing pattern in `variables.css`